### PR TITLE
feat: update readme

### PR DESCRIPTION
**Pull request by yukkysaito**
## Description
Discussed on [discord](https://discord.com/channels/953808765935816715/960543272483307582) and decided to update the readme.

## Related links
https://discord.com/channels/953808765935816715/960543272483307582

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,46 @@
-# autoware
+## Welcome to autoware! 👋
```
Comment (xmfcx) [inline README.md]:
```suggestion
## Welcome to Autoware! 👋
```
I think it is best to keep things consistent like:
- Autoware
- Autoware Core
- Autoware Universe

File: README.md [hunk 2]
```diff
@@ -1,5 +1,46 @@
...
+### Contribution guidelines
+
+- [There is no formal process to become a contributor](https://github.com/autowarefoundation/autoware-projects/wiki#contributors) - you can comment on any [existing issues](https://github.com/autowarefoundation/autoware.universe/issues) or make a pull request on any Autoware repository!
+  - Make sure to follow the [Contribution Guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/).
+  - Take a look at Autoware's [various working groups](https://github.com/autowarefoundation/autoware-projects/wiki#working-group-list) to gain an understanding of any work in progress and to see how projects are managed.
+- If you have any questions or ideas, feel free to start a discussion on [GitHub Discussions in autowarefoundation/autoware](https://github.com/autowarefoundation/autoware/discussions).
+
+### Useful resources
+
+- [Autoware documentation](https://autowarefoundation.github.io/autoware-documentation/main/) (also [open source](https://github.com/autowarefoundation/autoware-documentation)!)
```
Comment (xmfcx) [inline README.md]:
```suggestion
```

This is already mentioned above.

**Review by xmfcx — CHANGES_REQUESTED** [review]
Added some small suggestions

**Comment by yukkysaito** [comment]
@xmfcx Thank you for review. I fixed it :+1:

File: README.md (hunk 2 above)
Comment (yukkysaito) [reply README.md]: [50fcbfa](https://github.com/autowarefoundation/autoware/pull/151/commits/50fcbfafee925d12769cb0c5cb231c0c57a692f6)

File: README.md (hunk 1 above)
Comment (yukkysaito) [reply README.md]: [250e799](https://github.com/autowarefoundation/autoware/pull/151/commits/250e79922029933e96a5895dcba8779b5f2d7008)

**Review by xmfcx — APPROVED** [review]
Thanks!

**Pull request merged**